.github/workflows/devnet-deploys.yml

```diff
@@ -237,16 +244,51 @@ jobs:
         if: ${{ github.event.inputs.no_rebuild_images == 'false' }}
         run: |
           env
-          earthly-ci --no-output --push ./yarn-project+export-aztec-arch --DIST_TAG=${{ env.DEPLOY_TAG }}
-          earthly-ci --no-output --push ./yarn-project+export-aztec-arch --DIST_TAG=${{ github.sha }}
+          earthly-ci --no-output --push ./yarn-project+export-aztec-arch --DIST_TAG=${{ github.sha }} --ARCH=amd64
+          earthly-ci --no-output --push ./yarn-project+export-aztec-arch --DIST_TAG=${{ env.DEPLOY_TAG }} --ARCH=amd64
```

Comment:
Just a note for self (and others), the `+export-aztec-arch` target tags the image with the tag and arch combination